test/e2e/federated-service.go

```diff
@@ -41,19 +37,13 @@ import (
 )
 
 const (
-	UserAgentName = "federation-e2e-service-controller"
-	// TODO(madhusudancs): Using the same values as defined in the federated
-	// service controller. Replace it with the values from the e2e framework.
-	KubeAPIQPS   = 20.0
-	KubeAPIBurst = 30
+	SCUserAgentName = "federation-e2e-service-controller"
```

Comment:
updated federation user agent name to `federation-e2e`, and this can be reused by all federation e2e code